errtracker/errtracker.go

```diff
@@ -67,6 +69,106 @@ var (
 	timeNow  = time.Now
 )
 
+type reportsDB struct {
+	db *bolt.DB
```

Comment:
Ah, so you reverted to bolt? Is this for the locking we talked about?

Reply:
Yes.